packages/babel-polyfill/package.json

```diff
@@ -8,7 +8,7 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-polyfill",
   "main": "lib/index.js",
   "dependencies": {
-    "core-js": "^1.0.1",
+    "core-js": "^2.0.0",
```

Comment:
We can do major bumps on just the polyfill and runtime, which was the whole point in making them their own packages. I think their versioning should follow core-js